src/schemas/index.js

```diff
@@ -37,21 +60,13 @@ export const schema = {
       'type': 'link',
       'target': '/global-security/definitions'
     },
-    'definitions': {
-      'type': 'link',
-      'target': '/types'
-    },
     'info': {
       'type': 'link',
       'target': '/header/info'
     },
-    'contact': {
-      'type': 'link',
-      'target': '/header/contact'
-    },
-    'license': {
+    'externalDocs': {
       'type': 'link',
-      'target': '/header/license'
+      'target': '/header/externalDocs'
```

Comment:
`Linkfield`s just copy the value from another part of the form (not URLs). `externalDocs` is the name of the external documentation object in the OpenAPI spec.